docs/source/create/entities/zone-entity.rst

```diff
@@ -0,0 +1,164 @@
+| Cast Shadows           | This enables shadows, while still respecting the "Cast Shadows" property of other entities. |
++------------------------+---------------------------------------------------------------------------------------------+
+| Shadow Bias            | This controls shadow bias and needs to be fine tuned. Too low values can cause so called    |
+|                        | "shadow acne", while too high values can cause so called "peter panning".                   |
++------------------------+---------------------------------------------------------------------------------------------+
+| Shadow Max Distance    | Sets the maximum distance in meters from your view at which shadows will be shown.          |
++------------------------+---------------------------------------------------------------------------------------------+
+
+^^^^^^
+Skybox
+^^^^^^
+
+Skyboxes are used to define the background of a zone. There is two types of skybox images that can be used. The aspect ratio gets used to define the type of image. Images with a differing aspect ratio will not be displayed.
+
+Types:
+
+* Cubemaps have an aspect ratio of 4:3 and consist of 6 sides, which are projected onto a cube.
+
+.. image:: _images/cubemap-skybox.png
+
+* Equirectanglar images have an aspect ratio of 2:1 and are projected onto a sphere.
+
+.. image:: _images/equirectanglar-skybox.jpg
+
+^^^^^^^^^^^^^
+Ambient Light
+^^^^^^^^^^^^^
+
+Ambient lighting illuminates the view according to the brightness and color of a texture.
+Accepted properties are:
+
++------------------------+---------------------------------------------------------------------------------------------+
+| Property               | Description                                                                                 |
++========================+=============================================================================================+
+| Ambient Intensity      | This sets the intensity of the illumination.                                                |
++------------------------+---------------------------------------------------------------------------------------------+
+| Ambient Source         | This take the URL of a texture similar to the skybox. So similar to the skybox, that you    |
+|                        | can actually use the same texture as the skybox. One scenario where this could be useful,   |
+|                        | would be a spacestation, that is illuminated my multiple suns and reflections from          |
+|                        | different colored planets.                                                                  |
++------------------------+---------------------------------------------------------------------------------------------+
+
+^^^^
+Haze
+^^^^
+
+Haze creates fog inside a Zone. This can be used to make things look more unsettling, but is also useful for hiding distant geometry.
```

Comment:
```suggestion
Haze creates fog inside a Zone. This can be used to make things look more atmospheric but is also useful for hiding distant geometry.
```